### PR TITLE
TemplateSelector: Handle host not available

### PR DESCRIPTION
**Pull request by octaeder**
- show message when host is not available
- when network connection is available reopening a folder will load files

### CODE REVIEW

File: src/templateselector.cpp [hunk 1]
```diff
@@ -614,3 +625,15 @@ void TemplateSelector::openTemplateLocation()
 	}
 }
 
+void TemplateSelector::onRequestError()
+{
+    QNetworkReply *reply = qobject_cast<QNetworkReply *>(sender());
+    if (!reply) return;
+
+    QMessageBox::warning(this, tr("Browse template repository"),
```
Comment (sunderme) [inline src/templateselector.cpp:633]: blocking message boxes again
I would rather prefer to have a message in the tree view.

Comment (octaeder) [reply src/templateselector.cpp:633]: Thanks for pointing out this (again). I resolve this, see below

**Comment by octaeder** [comment]
I follow already available solution (item <loading...>. For example expand item Online Repository while network is not available. The you get:

image

When connection is available collapse and expand item Online Repository and you get:

image

**Comment by sunderme** [comment]
thanks

**Pull request merged**